oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/endpoint/NimbusJwtClientAuthenticationParametersConverter.java

```diff
@@ -189,6 +198,18 @@ else if (KeyType.OCT.equals(jwk.getKeyType())) {
 		return jwsAlgorithm;
 	}
 
+	/**
+	 * Sets the {@link Consumer} to be provided the {@link JwsHeader.Builder} and
```

Comment:
Can you update the javadoc to state...the `Consumer` is provided the `JwtClientAuthenticationContext`, which contains the `JwsHeader.Builder`...